docs/reference/commandline/cli.md

```diff
@@ -77,7 +77,7 @@ line:
 | `DOCKER_HIDE_LEGACY_COMMANDS` | When set, Docker hides "legacy" top-level commands (such as `docker rm`, and `docker pull`) in `docker help` output, and only `Management commands` per object-type (e.g., `docker container`) are printed. This may become the default in a future release, at which point this environment-variable is removed. |
 | `DOCKER_HOST`                 | Daemon socket to connect to.                                                                                                            |
 | `DOCKER_TLS_VERIFY`           | When set Docker uses TLS and verifies the remote. This variable is used both by the `docker` CLI and the [`dockerd` daemon](dockerd.md) |
-| `BUILDKIT_PROGRESS`           | Set type of progress output (`auto`, `plain`, `tty`) when [building](build.md) with [BuildKit backend](https://docs.docker.com/engine/reference/builder/#buildkit). Use plain to show container output (default `auto`). |
+| `BUILDKIT_PROGRESS`           | Set type of progress output (`auto`, `plain`, `tty`) when [building](build.md) with [BuildKit backend](https://docs.docker.com/build/buildkit/). Use plain to show container output (default `auto`). |
```

Comment:
This currently will be a 404; do you know when the new page will be up? https://docs.docker.com/build/buildkit/

Reply:
Yeah that's a chicken egg situation actually :sweat_smile:

When PRs on docker/cli are merged, I can kick ci again in https://github.com/docker/docs/pull/15958 so htmlproofer is happy and merge right away so we are synced.